environment.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
 - pydantic
 - preliz
 # NOTE: Keep minimum pymc version in sync with ci.yml `OLDEST_PYMC_VERSION`
-- pymc>=5.12.0,<5.16.0
+- pymc>=5.19.1
```

Comment:
@ricardoV94 so this should be `pymc>=5.20` ?

Reply:
Yes I didn't update yet

Reply:
Can you remove this: 

https://github.com/pymc-labs/pymc-marketing/blob/96848210a982279c50529ee828191cc9725b1c7f/pymc_marketing/mlflow.py#L166-L175

Reply:
I already removed it?

Reply:
Ah sorry I thought it was the mlflow from the requirements. Will do

Reply:
Sorry commented in wrong thread. Not related but was a backwards compat in the mlflow module